### PR TITLE
Eliminate the json flags of annex cmd for verbose flag

### PR DESCRIPTION
**Pull request by hkchekc**
This PR is based on achilleas's verbose-flag-tweaks branch. 

My 2 commits are majorly changing the annex-json prints. Now using a Global Boolean, the --json flag in annex command is replaced. Normal output is printed.

I am thinking we should add also the RawInput and RawOutput field for AnnexWhereisRes or AnnexStatusRes.

### CODE REVIEW

File: gincmd/common.go [hunk 1]
```diff
@@ -247,42 +247,20 @@ func verboseOutput(statuschan <-chan git.RepoFileStatus) (filesuccess map[string
...
-			_ = json.Unmarshal(outline, &output)
-			if !output.Success {
-				fmt.Printf("%s %s %s Progress:%d/%d(%s) FileKey:%s\r", output.Action.Command, output.Action.File, output.Action.Note,
-					output.ByteProgress, output.TotalSize, output.PercentProgress, output.Action.Key)
-			}
-		} else {
-			ro = stat.RawOutput
-			fmt.Printf("%s", ro)
+		//File Name
+		if stat.FileName != tmpfname {
```
Comment (achilleas-k) [inline gincmd/common.go]: Why is this necessary?  We don't want any custom/processed output in the raw output.

File: gincmd/common.go [hunk 2]
```diff
@@ -230,67 +243,72 @@ func printProgressOutput(statuschan <-chan git.RepoFileStatus) (filesuccess map[
...
+			fmt.Printf("File: %v\n", stat.FileName)
+			tmpfname = stat.FileName
 		}
+		//Raw Output
+		ro = stat.RawOutput
+		fmt.Printf("%s", ro)
```
Comment (achilleas-k) [inline gincmd/common.go]:
```suggestion
		fmt.Print(stat.RawOutput)
```

No need to use a separate name (`ro`) and no need to format since `stat.RawOutput` is a string.

File: gincmd/common.go (hunk 1 above)
Comment (hkchekc) [reply gincmd/common.go]: you mean the  filename printing?

Comment (achilleas-k) [reply gincmd/common.go]: Yes

**Comment by achilleas-k** [comment]
Merging because I need this.  Will fix small things later.

File: gincmd/common.go (hunk 1 above)
Comment (hkchekc) [reply gincmd/common.go]: you are right
